src/Features/CSharp/Portable/DesignerAttributes/CSharpDesignerAttributeService.cs

```diff
@@ -54,11 +54,6 @@ private IEnumerable<SyntaxNode> GetAllTopLevelTypeDefined(MemberDeclarationSynta
             return SpecializedCollections.EmptyEnumerable<SyntaxNode>();
         }
 
-        protected override bool ProcessOnlyFirstTypeDefined()
-        {
-            return true;
-        }
```

Comment:
both subclasses return true for this.  So we can just inline this value directly.  Because 'true' means 'break out of hte loop' we can take the loop below and remove it entirely.